build.gradle

```diff
@@ -1,67 +1,80 @@
 plugins {
-    id "java"
-    id "application"
-    id "idea"
-    id "org.sonarqube" version "3.3"
-    id "jacoco"
+	id "java"
+	id "application"
+	id "idea"
```

Comment:
As an aside, I'd recommend folks add this in their own Gradle config i.e. https://gitlab.com/jamietanna/dotfiles-arch/-/blob/master/java/home/.gradle/init.d/idea.gradle so we don't need to add unnecessary plugins for IDEs (especially as not everyone will use it!)